src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/IconUrlToImageCacheConverter.cs

```diff
@@ -146,7 +147,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
             return imageResult;
         }
 
-        public BitmapSource FinishImageProcessing(BitmapImage iconBitmapImage, Uri iconUrl, BitmapSource defaultPackageIcon)
+        private BitmapSource FinishImageProcessing(BitmapImage iconBitmapImage, Uri iconUrl, BitmapSource defaultPackageIcon)
```

Comment:
Is changing `public` method to `private` ok?

Reply:
This method should not be used as an API endpoint. That's why I changed to private. Normally, in VS-NuGet components, we are more permissive on API changes, because those assemblies are not intended to be consumed outside Visual Studio.

What do you think, @zivkan ? Should I go back to `public`?

Reply:
I will revert it, since method visibility will not affect the core PR bugfix

Reply:
`NuGet.PackageManagement.UI` is not part of the NuGet SDK (hence changing visibility did not require a change to a `PublicAPI.[Un]shipped.txt` file. So, it really doesn't matter. It's code that's only used internally.

From an idealistic point of view, it's better for everything to be internal or private, where possible. But there isn't a realistic way for other VS components to access our assembly directly, so it really doesn't matter (just like public APIs in a console app).